### PR TITLE
FireEye ETP bug

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-20981

## Description
Added support for status messages as a list of multiple statuses rather than just one.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FireEyeETP/Integrations/FireEyeETP/FireEyeETP.yml [hunk 1]
```diff
@@ -35,8 +35,21 @@ configuration:
 - display: Messages status. All messages with the status specified will be imported as incidents.
```
Comment (dorschw) [inline Packs/FireEyeETP/Integrations/FireEyeETP/FireEyeETP.yml:35]:
```suggestion
- display: Message statuses to import
  additionalinfo: All messages with a status specified here will be imported as incidents.
```

File: Packs/FireEyeETP/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "FireEye ETP",
     "description": "FireEye Email Threat Prevention (ETP Cloud) is a cloud-based platform that protects against advanced email attacks.",
     "support": "xsoar",
-    "currentVersion": "1.1.0",
+    "currentVersion": "1.1.1",
```
Comment (dorschw) [inline Packs/FireEyeETP/pack_metadata.json:5]: worth a `1.2.0`

**Review by dorschw — COMMENTED** [review]
Nice!

File: Packs/FireEyeETP/ReleaseNotes/1_2_0.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### FireEye ETP
+- Added support for specifying multiple status values(multi-select) for messages for fetching messages as incidents.
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeETP/ReleaseNotes/1_2_0.md:4]:
```suggestion
- Added support for specifying multiple status values (multi-select) for fetching messages as incidents. 
```

**Comment by ShirleyDenkberg** [comment]
@dorschw Doc review completed.

**Review by dorschw — APPROVED** [review]

**Pull request merged**